lib/src/components/form/FormInput.js

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import { Input } from 'lib';
+import { FormContext, statuses } from './Form';
+
+export function FormInput({ children, ...rest }) {
```

Comment:
🔴 

It would be nice to enforce a bit of consistency here. Every form input needs an `id` and a `name`. I hate to say this but maybe prop-types would help here to make sure that the input _always_ gets those.